docs/spark-connector/spark-catalog-hive.md

```diff
@@ -51,3 +51,17 @@ INSERT OVERWRITE TABLE employees PARTITION(department='Marketing') VALUES (3, 'M
 
 SELECT * FROM employees WHERE department = 'Engineering';
 ```
+
+
+## Catalog properties
+
+| Gravitino catalog property name | Spark catalog property name | Description                | Since Version |
+|---------------------------------|-----------------------------|----------------------------|---------------|
+| `metastore.uris`                | `hive.metastore.uris`       | Hive metastore uri address | 0.5.0         |
+
+Catalog properties with prefix `spark.bypass.` are passed to Spark Hive connector. For example, using `spark.bypass.config.resources` to pass the `config.resources` to the Spark Hive connector.
```

Comment:
So should we use "Gravitino catalog property name" or "Spark catalog property name", you brings new concepts without any explanation, how do users use it?

Reply:
replace with `Gravitino catalog property names`

Reply:
Again, as I mentioned here, you add two tables in here and iceberg for a bunch of properties without explaining what are they and how user uses them, so what actually do you want to deliver to users?

Reply:
add some description, please help to review again